app/components/note.module.css

```
@@ -1,14 +1,31 @@
 .note {
   border: solid #000;
   border-width: 0 0 0.5px 0;
-  padding: 0 0.5rem;
+  padding: 0.5rem;
 }
 
 .accountNameContainer {
   font-size: small;
-  span {
-    font-weight: lighter;
-    font-size: smaller;
-    margin-left: 0.5rem;
+  display: flex;
+  align-items: center;
+  overflow: hidden;
```

```suggestion
  overflow: hidden;
  text-overflow: ellipsis;
```
テキストのはみ出しを抑制するのであれば, [`text-overflow`](https://developer.mozilla.org/ja/docs/Web/CSS/text-overflow) を指定したほうがよいかもしれません.

---

すいません忘れてました, ユーザー名表示のように折り返しされたくないところは `white-space: nowrap;` も必要かと思います.